Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -3987,6 +3991,21 @@ void CClientPed::_ChangeModel()
             _CreateModel();
         }
 
+        // ReAttach satchels
+        static CClientProjectileManager* pProjectileManager = m_pManager->GetProjectileManager();
```

Comment:
I would avoid using static here. GetProjectileManager is cheap and there's no need in caching it.